azurerm/internal/services/compute/dedicated_host_resource.go

```diff
@@ -172,7 +172,7 @@ func resourceArmDedicatedHostRead(d *schema.ResourceData, meta interface{}) erro
 		return err
 	}
 
-	group, err := groupsClient.Get(ctx, id.ResourceGroup, id.HostGroup)
+	group, err := groupsClient.Get(ctx, id.ResourceGroup, id.HostGroup, compute.InstanceView)
```

Comment:
`InstanceView` returns a different response from the API - presumably this wants to be an empty string?

Reply:
DOne